FirebaseAuth/Sources/Swift/Utilities/AuthWebUtils.swift

```diff
@@ -77,6 +77,29 @@ import Foundation
     return false
   }
 
+  /** @fn extractDomain:urlString
+   @brief Strips url of scheme and path string to extract domain name
+   @param urlString URL string for domain
+   */
+  static func extractDomain(urlString: String) -> String? {
+    var domain = urlString
+
+    // Check for the presence of a scheme (e.g., http:// or https://)
+    if domain.prefix(7).caseInsensitiveCompare("http://") == .orderedSame {
+      domain = String(domain.dropFirst(7))
+    } else if domain.prefix(8).caseInsensitiveCompare("https://") == .orderedSame {
+      domain = String(domain.dropFirst(8))
+    }
+
+    // Remove trailing slashes
+    domain = (domain as NSString).standardizingPath as String
```

Comment:
nit: any reason not to do this on line 85?

Reply:
Yes. That was my first implementation, but it consolidates the "//" after `http:` to a single slash and would make the code less readable.

Reply:
ah, makes sense. Thanks for the explanation.